yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -330,9 +373,6 @@ private object YarnClusterDriver extends Logging with Matchers {
 }
 
 private object YarnClasspathTest extends Logging {
-
-  var exitCode = 0
```

Comment:
Not removing this will break test, because System.exit(0) will cause app end with status 'FAILED' (after changed in v2.0 by https://github.com/apache/spark/commit/d89c71417c384e1a2820bd80b67c26e11405aebc)
So I should remove this System.exit().